main.tf

```diff
@@ -161,13 +160,13 @@ resource "spacelift_policy_attachment" "trigger_administrative" {
   count = var.external_execution || var.administrative_trigger_policy_enabled == false ? 0 : 1
 
   policy_id = join("", spacelift_policy.trigger_administrative.*.id)
-  stack_id  = data.spacelift_current_stack.administrative[0].id
```

Comment:
We'll always need to grab the current stack id w/ the new logic.. I don't think this will cause any issues but my memory is a little fuzzy about this piece. There's also some other logic that has been added that may cause issues if it's not always enabled (e.g. around the policy assignments elsewhere in this file).